scripts/src/admin/services.ts

```diff
@@ -693,18 +693,23 @@ export async function changeOffer(body: ChangeOfferData, params: { offer_id: str
 	await offer.save();
 }
 
-type UpdateAppConfigRequest = Request & {
-	body: ApplicationConfig
-};
+type AppConfig = ApplicationConfig & { [limits: string]: number };
+type UpdateAppConfigRequest = Request & { body: AppConfig; };
 
 export const updateAppConfig = async function(req: UpdateAppConfigRequest, res: Response) {
+	const config: AppConfig = req.body;
+	const isLimitsNumbers = (configObj: AppConfig, limitName: string) => typeof configObj.limits[limitName as keyof AppConfig["limits"]] === "number";
+	if (!config.limits || !Object.keys(config.limits).every(isLimitsNumbers.bind(null, config))) {
+		res.status(400).send("Config data is incorrect");
```

Comment:
I'd change `incorrect` to `invalid`. Incorrect deals with wrong numbers, invalid relates to format. This might sound like mere semantics but think about the user, you want Tom to know it's not that he entered bad numbers, you want to tell him "something is wrong with the format of what you've submitted, have another look or go bother Lior about it"